src/metabase/config.clj

```diff
@@ -144,6 +144,10 @@
   (when-let [user-json (env/env :mb-user-defaults)]
     (json/parse-string user-json true)))
 
+(def ^:const audit-db-id
+  "ID of Audit DB which is loaded when running an EE build. ID is placed in OSS code to facilitate permission checks."
+  13371337)
```

Comment:
I've kept this here but not removed the other location of this var for now, to avoid further merge conflicts. That can be part of the cleanup after the feature branch gets merged.